src/App.tsx

```diff
@@ -2,98 +2,18 @@
 import React from 'react';
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
+import { Movie } from './types/Movie';
 
-export const App: React.FC = () => (
+type Props = {
+  movies: Movie[];
+};
```

Comment:
Why do you need this props?

Reply:
You`re right, this props is really redundant, my mistake)